modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -302,6 +302,9 @@ class ApiClient:
         elif isinstance(obj, list):
             return [self.sanitize_for_serialization(sub_obj)
                     for sub_obj in obj]
+        elif isinstance(obj, set):
+            return {self.sanitize_for_serialization(sub_obj)
+                    for sub_obj in obj}
```

Comment:
Not useful for now. `Annotated[List, Field(unique_items=True)]` should be replaced by `Annotated[Set]`, but this has further implications (sets lose the ordering, not serializable as JSON automatically, etc.)

```suggestion
```

Reply:
thanks for the heads-up. later we can add some tests in https://github.com/OpenAPITools/openapi-generator/blob/master/samples/openapi3/client/petstore/python/tests/test_api_client.py#L159 to cover `set`

Reply:
Actually, I wanted to remove this block of code, as it's unused right now.
There's a comment in the related part of the Java code:

```java
        private PythonType arrayType(IJsonSchemaValidationProperties cp) {
            PythonType pt = new PythonType();
            if (cp.getMaxItems() != null) {
                pt.constrain("max_items", cp.getMaxItems());
            }
            if (cp.getMinItems()!= null) {
                pt.constrain("min_items", cp.getMinItems());
            }
            if (cp.getUniqueItems()) {
                // A unique "array" is a set
                // TODO: pydantic v2: Pydantic suggest to convert this to a set, but this has some implications:
                // https://github.com/pydantic/pydantic-core/issues/296
                // Also, having a set instead of list creates complications:
                // random JSON serialization order, unable to easily serialize
                // to JSON, etc.                                                                                                                                                                                                                                                
                //pt.setType("Set");
                //typingImports.add("Set");
                pt.setType("List");
                typingImports.add("List");
            } else {
                pt.setType("List");
                typingImports.add("List");
            }
            pt.addTypeParam(getType(cp.getItems()));
            return pt;
        }
```